### ISSUE TITLE
commit.labels and refactoring inconsistencies

### ISSUE THREAD

**Issue by shehan** [smartshark/smartshark.github.io#29]
In the `commit` collection there are some records that do not have a `refactoring_codebased` label even though there is a record in the `refactoring` collection for that specific commit.

Example:
Query the `commit` and `refactoring` collections for the Commit.id = "5e84c0b6770ce37ee9d708b3"
There are 21 records in the `refactoring` collection
However, in the `commit` collection, the `labels` field is empty. Shouldn't the `labels` field be populated?

A few more commit id's with exhibiting this phenomenon:
5e84c208770ce37ee9d771dd
5e84c0f9770ce37ee9d71a89

I am using Smartshark version 2.1 (full db)

**Comment by atrautsch** [comment]
You are right, the labels are missing and should be there. There was an error with labelSHARK which writes the labels which I missed in the collection phase. Project maven is the only one that is impacted by this.

If you like, you can import the labels separately. 
```bash
wget http://141.5.100.155/commit_labels_maven.json.gz
gunzip commit_labels_maven.json.gz
mongoimport --db=smartshark_2_1 --collection=commit --mode=merge --upsertFields=_id commit_labels_maven.json
```
